.circleci/windows/msys2_prepare.bat

```diff
@@ -0,0 +1,4 @@
+@echo on
+git config --global --unset url."ssh://git@github.com".insteadOf "https://github.com"
```

Comment:
CircleCI by default has set this in git config, but it prevents to download from public repos (specifically styrene) without a key